ui/app/AppLayouts/Profile/popups/ChangePasswordModal.qml

```diff
@@ -24,9 +24,8 @@ StatusModal {
 
     function onChangePasswordResponse(success, errorMsg) {
         if (success) {
-            if (Qt.platform.os === "osx" && localAccountSettings.storeToKeychainValue !== Constants.storeToKeychainValueNever) {
-                localAccountSettings.storeToKeychainValue = Constants.keychain.storedValue.store;
-                root.privacyStore.storeToKeyChain(d.passwordProcessing);
+            if (Qt.platform.os === "osx" && localAccountSettings.storeToKeychainValue !== Constants.keychain.storedValue.never) {
+                localAccountSettings.storeToKeychainValue = Constants.keychain.storedValue.notNow;
```

Comment:
@mprakhov maybe we could remove password from the keychain here?

Reply:
does it really make sense, if we will force the user to restart the application and update the pass in the keychain during the next application start?